src/fides/ctl/core/export.py

```diff
@@ -429,7 +429,7 @@ def build_joined_dataframe(
     return joined_df
 
 
-def export_datamap(
+def export_datamap(  # pylint: disable=too-many-locals
```

Comment:
This should probably just be globally disabled in `pyproject.toml`. Thoughts?

Reply:
I have never see us not ignore this when it shows up so it makes sense to me to globally disable this. I'll add it.